javascript/javascript1/week2/studentmanager.js

```diff
@@ -0,0 +1,38 @@
+const class07Students = [];
+const queen='Margrethe II';
+function addStudentToClass(studentName) {
+  if (!class07Students.includes(queen)){
+      class07Students.push(queen);
+      console.log(`queen ${queen} has been added to the the class07`);
+} if  (class07Students.length>6){
+    console.log("Cannot add more students to class 07");
+      } else if (class07Students.length){
```

Comment:
Check includes https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes
```suggestion
      } else if (class07Students.includes(studentName)){
```